lib/glific/saas/onboard.ex

```diff
@@ -232,6 +234,26 @@ defmodule Glific.Saas.Onboard do
 
   defp process_on_submission(result, _org, _registration), do: result
 
+  @doc """
+  Updates password_hash field of passed org_id with hashed password generated via Glific.Password
+  """
+  @spec update_process_on_submission(any()) :: {:error, binary()} | {:ok, <<_::232>>}
+  def update_process_on_submission(org_id) do
```

Comment:
Can we rename this function, since it focus on updating the password of NGO account